db/data_migrations/20250226173258_update_rejected_reasons_data.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class UpdateRejectedReasonsData < ActiveRecord::DataMigration
+  disable_ddl_transaction!
```

Comment:
Is this necessary?

Reply:
good spot - it came with the autogenerated file and I forgot to delete it, I don't believe it is needed so will remove it.